spacy/sklearn.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+from sklearn.base import BaseEstimator, TransformerMixin
+import spacy
+
+
+__all__ = [
+    'SpacyTransformer',
+    'DOC', 'VECTOR', 'SENTENCES', 'ENTITIES', 'NOUN_CHUNKS',
+    'SENTIMENT', 'TOKENS'
+    ]
+
+
+DOC = 'self'
+VECTOR = 'vector'
+SENTENCES = 'sents'
+ENTITIES = 'ents'
+NOUN_CHUNKS = 'noun_chunks'
+SENTIMENT = 'sentiment'
+TOKENS = 'tokens'
+
+
+class SpacyTransformer(BaseEstimator, TransformerMixin):
+
+    def __init__(self,
+                 model=spacy.en.English(),
```

Comment:
In the beginning, I had the wrapper take the language string (i.e. `en`, `fr`, etc.) as the argument and initialize the model in as 

    self.model_ = spacy.load(self.language)

in the `fit` method, but the long model loading time for each `fit` call made me think that doing it this way would be more efficient.

If this is a concern, perhaps we could go back to that first method and leverage the `fit_params` so the user can provide a loaded model if they want.

    def fit(self, X=None, y=None, **fit_params):
        if 'model' in fit_params:
            self.model_ = fit_params['model']
        else:
            self.model_ = spacy.load(self.language)